content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,86 +10,91 @@ further_reading:
+## Configuring the Datadog Exporter
 
-Here is an example trace pipeline configured with an `otlp` receiver, `batch` processor and `datadog` exporter:
+To use the Datadog Exporter, simply add it to your [OpenTelemetry Collector configuration][4]. Here is a bare-bones configuration file that is ready-to-use as soon as you have set your Datadog API key as the DD_API_KEY environment variable:
 
-```
+```yaml
 receivers:
   otlp:
     protocols:
-      grpc:
       http:
+      grpc:
 
 processors:
   batch:
 
 exporters:
   datadog:
-    host_metadata:
-      tags:
-        - example:tag
     api:
-      key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
-      site: {{< region-param key="dd_site" code="true" >}}
+      key: ${DD_API_KEY}
 
 service:
   pipelines:
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [datadog]
     traces:
       receivers: [otlp]
       processors: [batch]
       exporters: [datadog]
 ```
 
-## Environment specific setup
+The above configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries via HTTP and gRPC and sets up a [batch processor][5] which is mandatory for any non-development environment.
 
-### Host
+### Advanced configuration
 
-1. Download the appropriate binary from [the project repository latest release][9].
+A fully documented example configuration file can be found [here][6], illustrating all possible configuration options for the Datadog Exporter. There may be other options relevant to your deployment, such as `site` or `host_metadata`.
 
-2. Create a `otel_collector_config.yaml` file. [Here is an example template](#ingesting-opentelemetry-traces-with-the-collector) to get started. It enables the collector's OTLP Receiver and Datadog Exporter.
+### Host name resolution
 
-3. Run the download on the host, specifying  the configration yaml file set via the `--config` parameter. For example:
+The host name that OpenTelemetry signals are tagged with is obtained based on the following sources in order, falling back to the next one if the current one is unavailable or invalid:
 
-      ```
-      otelcontribcol_linux_amd64 --config otel_collector_config.yaml
-      ```
+1. From [resource attributes][7] (such as for example `host.name`, but many others are supported).
+2. The `hostname` field in the exporter configuration.
+3. Cloud Provider API.
+4. Kubernetes host name.
+5. Fully Qualified Domain Name.
+6. Operating System host Name.
 
-### Docker
+
+## Configuring your application
+
+There are a few things you could do to make sure you get better metadata for traces and a smooth integration with Datadog.
+
+### Resource Detectors
+
+The OpenTelemetry specification requires that SDKs implement [resource detectors][8] for common platforms and environments (such as Docker, Kubernetes, EKS, ECS, GKE, etc.). For example, the Go repository [has them available][9] for AWS and GCP.
+
+We strongly recommend using these if available in your SDK.
```

Comment:
We have so far only recommended and thoroughly tested the OpenTelemetry Collector resource detection processor, and not invidual language SDK detectors. While they should work, I think we should recommend using what we know for certain works: the resource detection processor.

Reply:
The Container ID one does work, unless they are using cgroups2. It's quite valuable to have the container ID. I think we should recommend using both the [SDK detectors](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/detectors) and the [resource detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor) in the Collector. 

Does this change anything?

Reply:
I am fine recommending that one 👍